### DISCUSSION TITLE
Can FlyBy11 create a modified ISO with the sever method?

### DISCUSSION THREAD

**Discussion by Peter-Go** [builtbybel/FlyOOBE#15] [General]
Ca FlyBy11 create an ISO to use in something like Ventoy using the server setup method?

**Comment by Belim** [comment]
Yep, for this, just go to Media Control, option at bottom **I want to create Windows 11 installation media** There, you can directly create the ISO with FlyBy11, or use Microsoft’s Media Creation Tool, and finally apply the bypass patch using the new **Apply USB Compatibility Patch** feature introduced in version 0.15

**Reply by adrianp74** [reply]
Doesn't work, it says "Please enter a valid Disk Number". The Device ID drop down is disabled and so is the Device Letter. I selected an ISO file that I already had.

**Reply by Belim** [reply]
then there’s probably no USB stick plugged in?

**Comment by Peter-Go** [comment]
Would it be possible to create an iso from the output so that it can be
used in things like ventoy and iventoy?

**Reply by ThomasRoux51** [reply]
Hello,

I followed this instruction from Media Creator + USB apply patch which 'worked' to make a bootable USB key then I boot from the key on the PC i wanted to install (from scratch) the Win11 but got stuck at the very beginning of the install saying that the PC doesn't match the minimum configuration specs...

So is there something I did wrong ? or is there a specific method to create a bootable USB key to install Win11 from scratch on a PC that currently does not meet the minimum required config (no TPM 2, ..) ?

**Comment by ecomcodeLab** [comment]
i dont understand i want to make a iso that have the fixes directly inside the file but i cant see this option . please help

**Comment by HaukeMA** [comment]
I don't get it either. Why do I have to have a USB stick plugged in if I just want to patch the ISO?

**Comment by Cpt-KeksMonster** [comment]
To me, "Modify ISO" suggests that the input is an unmodified ISO file and the output is a modified ISO file.

However, Flyoobe actually requires a USB installation stick that contains the extracted contents of an ISO file (and must be writable, unlike a mounted ISO). It then modifies this USB stick — which is much more comparable to what Rufus does — BUT with an existing installation medium. In my view, this is not a modification of an ISO file at all.
